src/wizards/newsletters/views/settings/index.js

```diff
@@ -393,15 +393,15 @@ export const SubscriptionLists = ( { lockedLists, onUpdate, initialProvider } )
 	);
 };
 
-const Newsletters = () => {
+const NewslettersSettings = () => {
 	const [ { newslettersConfig }, updateConfiguration ] = hooks.useObjectState( {} );
 	const [ initialProvider, setInitialProvider ] = useState( '' );
 	const [ lockedLists, setLockedLists ] = useState( false );
 	const [ authUrl, setAuthUrl ] = useState( false );
 
 	return (
 		<>
-			<NewspackNewsletters
+			<Settings
```

Comment:
```suggestion
			<h1>{ __( 'Newsletter Settings', 'newspack-plugin' ) }</h1>
			<Settings
```

Reply:
It feels like we're saying the same thing 3 times:

<img width="471" alt="image" src="https://github.com/user-attachments/assets/5f8aecec-db28-47e3-b834-58cd9811dbf5">

It's how it's proposed in the design, but since the design screen doesn't have the wizard header, maybe it slipped by. @thomasguillot could you confirm that this is the intended result?

Reply:
I think it's ok if we repeat things and it's to have some sort of consistency with the other wizards. We could probably the heading to simply "Settings"

Reply:
Which one?

Reply:
Added in ac26dbc1df32e3810831bcbf0211853694332724

Reply:
> Like this?

Yes